### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by neuweltgeld**
Fixes folder name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ export PATH=$GOBIN:$PATH
 With this method, you generate the binary manually (need Go installed), downloading and compiling the source code:
 
 ```bash
-git clone https://github.com/Layr-Labs/eigenlayer-cli.git
+git clone https://github.com/Layr-Labs/eigenlayer-cli.git && mv eigenlayer-cli eigenlayer
```
Comment (shrimalmadhur) [inline README.md:82]: actually better would be to change `cd eigenlayer` to `cd eigenlayer-cli` in next line @neuweltgeld

Comment (neuweltgeld) [reply README.md:82]: if ``eigenlayer-cli`` is to be used as folder name, PATH variable also be changed
````
# Using Go
sudo cp $GOPATH/bin/eigenlayer /usr/local/bin/

# Build from source
sudo cp eigenlayer/build/eigenlayer /usr/local/bin/
````

Comment (shrimalmadhur) [reply README.md:82]: yea - good catch - feel free to change that too. It's better than renaming the repo.

Comment (neuweltgeld) [reply README.md]: > yea - good catch - feel free to change that too. It's better than renaming the repo.

done √

**Review by shrimalmadhur — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shrimalmadhur) [reply README.md]: Thank you!

**Pull request merged**